plugin.json

```diff
@@ -29,6 +29,38 @@
         "type": "generated",
         "help_text": "The secret used to authenticate the webhook to Mattermost.",
         "regenerate_help_text": "Regenerates the secret for the webhook URL endpoint. Regenerating the secret invalidates your existing JIRA integrations."
+      },
+      {
+        "key": "SubscriptionEditorsMattermostRoles",
+        "display_name": "Subscription Editors Mattermost Roles",
```

Comment:
@aaronrothschild are these names obvious on first read?

Reply:
No, I have no idea what this means after reading it 4 times.  

Where does this text show up?

Reply:
This would appear in the System Console > Plugins > Jira settings page

Reply:
suggest: RolesAllowedToEditJiraSubscriptions ?

Reply:
I would love suggestions for a better name. This was @levb suggestion above.

Reply:
I like Chris P' suggestions about `GroupsAllowedToEditJiraSubscriptions` and `RolesAllowedToEditJiraSubscriptions`, in addition to making it clear it's for "Jira Groups" and "Mattermost Roles". (cc @aaronrothschild )

@wiersgallak tagging in case you have thoughts / feedback to share. In summary, we are adding two settings in System Console > Jira, which enable a SysAdmin to control who can subscribe Jira projects to Mattermost channels. By default it is Mattermost SysAdmins only, but you can enable Mattermost users to subscribe based on
 - their role in MM (sysadmin, team admin, channel admin, everyone)
 - their group in Jira